### ISSUE TITLE
The names in the target that started with prefix "set" are not recognized correctly

### ISSUE THREAD

**Issue by mxmind** [mapstruct/mapstruct#1926]
- [x] Is this an issue (and hence not a question)? 

If this is a question how to use MapStruct there are several resources available.
- Our reference- and API [documentation](http://mapstruct.org/documentation/reference-guide/).
- Our [examples](https://github.com/mapstruct/mapstruct-examples) repository (contributions always welcome)
- Our [FAQ](http://mapstruct.org/faq/)
- [StackOverflow](https://stackoverflow.com), tag MapStruct 
- [Gitter](https://gitter.im/mapstruct/mapstruct-users) (you usually get fast feedback)
- Our [google group](https://groups.google.com/forum/#!forum/mapstruct-users)

**Comment by mxmind** [comment]
The names in the target that started with prefix "set" are not recognized correctly. The example is "settlement". See the attached picture for more details.  


![me… 2019-09-26 01-34-59](https://user-images.githubusercontent.com/1923314/65644954-63c43100-dffe-11e9-85af-3a7cc17ef9bd.png)  

Then if the suggested filed name is set, this following diagnostic error raised:  

```
No property named "tlement" exists in source parameter(s). Did you mean "settlement"?
    AddressDocument from(EhealthAddressDto source);
```

Otherwise, if the correct fieldName is set, the error is still present, but with a different explanation:  

```
Unknown property "settlementType" in result type io.h**.m**.acm.spi.model.ehealth.EhealthAddressDto. Did you mean "null"?
    EhealthAddressDto to(AddressDocument source);
```

**Comment by filiphr** [comment]
This is a duplicate of https://github.com/mapstruct/mapstruct/issues/1799. It has been fixed on master and for the next 1.3.1 bug fix release.
